src/components/ContinuationIn/ContinuationAuthorization.vue

```diff
@@ -138,26 +64,67 @@
             title="Expiry Date (Optional)"
             :nudgeRight="40"
             :nudgeTop="85"
+            hint="The date the authorization expires."
+            :persistentHint="true"
             :initialValue="authorization.expiryDate"
             :inputRules="getShowErrors ? expiryDateRules: []"
             :minDate="getCurrentDate"
             @emitDateSync="authorization.expiryDate = $event"
           />
         </v-col>
       </v-row>
+    </v-form>
+
+    <v-divider class="mt-6 mb-8" />
+
+    <header>
+      <p>
+        Upload documents that support proof of authorization from your home jursidiction.
+      </p>
+
+      <ul>
+        <li>
+          Use a white background and a legible font with contrasting font colour
+        </li>
+        <li>
+          PDF file type (maximum 30 MB file size)
+        </li>
+      </ul>
+    </header>
+
+    <div class="pt-4" />
+
+    <v-row
+      v-for="(num, index) in numUploads"
+      :key="authorization.files[index]?.fileKey"
+      class="upload-file-row mt-4 mb-n2"
```

Comment:
Would it be better to space in a way to avoid using negative margins? It is advised to keep away from using negative spacing when possible right Sev?

Reply:
I usually advise against it because there's usually a better way (like "hide-details=auto"). But that's not the problem here. I simply wanted to tighten up the whitespace between these multiple file upload rows while keeping the top margin on each.

Without:
![image](https://github.com/bcgov/business-create-ui/assets/10002889/09e0ab75-cc01-40af-8bc6-bca5b5f5ee18)

With:
![image](https://github.com/bcgov/business-create-ui/assets/10002889/06659e6e-15f5-4f55-96c5-7a090900b639)

I guess my judgement says to not use negative margin to affect the next component, but it's OK to fix the current component.

Reply:
That makes so much sense to me. All right, sounds good!